tests/test_analysis/test_outliers/test_density_outliers.py

```diff
@@ -16,9 +16,7 @@ def simple_window() -> np.array:
 
 def test_const_ts(const_ts_anomal):
     anomal = get_anomalies_density(const_ts_anomal)
-    assert {"segment_0", "segment_1"} == set(anomal.keys())
-    for seg in anomal.keys():
-        assert len(anomal[seg]) == 0
+    assert len(anomal) == 0
```

Comment:
I doesn't really understand why the answer has changed here.

Reply:
This PR proposes to store only segments with detected anomalies. So in this case empty dict should be returned.